lib/ansible/modules/remote_management/hpilo/hponcfg.py

```diff
@@ -40,6 +40,20 @@
   minfw:
     description:
     - The minimum firmware level needed
+    required: false
+  hponcfg:
```

Comment:
This should become `executable`, which is the standard parameter name for this kind of functionality in Ansible.